### PR TITLE
TA 2023-05-15-preview metadata fix

### PR DESCRIPTION
**Pull request by quentinRobinson**
# Data Plane API - Pull Request

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*API Spec engagement record*](https://aka.ms/azsdk/onboarding/restapischedule).

* Link to API Spec engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [x] a public preview
- [ ] GA release

### Change Scope
This Change is to change the swagger definition to match the actual output of the API.

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azsdk/onboarding/restapischedule) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azsdk/onboarding/restapischedule).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

File: dev/cognitiveservices/data-plane/Language/analyzetext.json [hunk 1]
```diff
@@ -2766,6 +2766,20 @@
...
       "description": "The reference point that the ordinal number denotes."
     },
+    "RangeInclusivity": {
+      "type": "string",
+      "enum": [
+        "noneInclusive",
+        "leftInclusive",
+        "rightInclusive",
+        "leftRightInclusive"
+      ],
```
Comment (heaths) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:2776]: While REST API guidelines to recommend camelCase for enums, this is not idiomatic of your service that uses PascalCase for enums. I recommend sticking with PascalCase.

File: dev/cognitiveservices/data-plane/Language/analyzetext.json [hunk 2]
```diff
@@ -2810,7 +2824,7 @@
             "InformationMetadata",
             "TemperatureMetadata",
             "CurrencyMetadata",
-            "NumberRangeMetadata",
+            "NumericRangeMetadata",
```
Comment (heaths) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:2827]: Has this already GA'd? If so, this is breaking and needs to go through the breaking change board.

File: dev/cognitiveservices/data-plane/Language/analyzetext.json [hunk 3]
```diff
@@ -3202,15 +3224,15 @@
         }
       }
     },
-    "NumberRangeMetadata": {
+    "NumericRangeMetadata": {
```
Comment (heaths) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:3227]: This is potentially SDK-breaking if it already GA'd.

File: dev/cognitiveservices/data-plane/Language/analyzetext.json (hunk 2 above)
Comment (heaths) [reply dev/cognitiveservices/data-plane/Language/analyzetext.json:2827]: Yes, this already GA'd: https://github.com/Azure/azure-rest-api-specs/pull/24768/checks?check_run_id=14957945150. Please revert. Given it's just a name change, I don't think this rises to the level of a required breaking change.

**Review by heaths — CHANGES_REQUESTED** [review]

File: dev/cognitiveservices/data-plane/Language/analyzetext.json (hunk 2 above)
Comment (quentinRobinson) [reply dev/cognitiveservices/data-plane/Language/analyzetext.json:2827]: Metadata is not GA'd it is something we introduced this preview version. I dont know why we are getting this false positive

File: dev/cognitiveservices/data-plane/Language/analyzetext.json (hunk 3 above)
Comment (quentinRobinson) [reply dev/cognitiveservices/data-plane/Language/analyzetext.json:3227]: Metadata is not GA'd it is something we introduced this preview version.

File: dev/cognitiveservices/data-plane/Language/analyzetext.json (hunk 1 above)
Comment (quentinRobinson) [reply dev/cognitiveservices/data-plane/Language/analyzetext.json]: Fixed!

File: dev/cognitiveservices/data-plane/Language/analyzetext.json [hunk 4]
```diff
@@ -2769,10 +2769,10 @@
     "RangeInclusivity": {
       "type": "string",
       "enum": [
-        "noneInclusive",
-        "leftInclusive",
-        "rightInclusive",
-        "leftRightInclusive"
+        "NoneInclusive",
```
Comment (heaths) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:2772]: Can I also assume the service code is/will be updated?

Comment (quentinRobinson) [reply dev/cognitiveservices/data-plane/Language/analyzetext.json:2772]: yes, we have a fix for it.

**Comment by quentinRobinson** [comment]
The breaking changes in the PR are false positives. The metadata was introduced in this preview version and has never been GA'd

**Comment by heaths** [comment]
The only breaking changes I see are an old one that was intentional and previously approved i.e., adding `type` (of either "string" or "object" as needed) to satisfy some other validation failures.

However, please add "Inclusivity" to the dictionary to resolve the spellcheck error once and for all.

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
The breaking changes around adding `type: string` and `type: object` was previously approved, nor consequential to our code generation that hasn't already been solved.

**Pull request merged**